Clinical-Genomics/cg
# first commit - adding invoices for microbial

This PR adds Microbial invoicing to the Invoicing tool

I have just added code following the code layout that already is in place for samples and pools. 
I intend to update the structure a bit, add dokumentation to all functions in the code etc. in a separate PR.

How to test:
0. install on stage:
sh ~/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh microbial
test it [here](https://clinical-api-stage.scilifelab.se/invoices/new/Microbial)

1. select: New Invoice - Microbial. Only customers with microbial samples should show up in the drop down.
2. select customer. - Only microbial samples should show up
3. select samples. - Should give no errors
4. create invoice. - Should give no errors

- [x] functionality tested by @emiliaol 
- [x] code approved for deploy by @ingkebil 

This is a minor because the changes are backwards compatible.
